stack/values.schema.json

```diff
@@ -793,32 +794,33 @@
           "type": "array"
         }
       },
-      "type": "object"
+      "additionalProperties": false
     },
     "jobs": {
       "description": "Jobs to deploy",
+      "type": "object",
       "patternProperties": {
         "^.*$": {
           "$ref": "#/properties/global",
-          "type": "object"
+          "type": "object",
+          "additionalProperties": false
         }
       },
-      "properties": {},
-      "type": "object"
+      "additionalProperties": false
     },
     "services": {
       "description": "Services to deploy",
+      "type": "object",
       "patternProperties": {
         "^.*$": {
           "$ref": "#/properties/global",
           "type": "object",
+          "additionalProperties": false,
           "unevaluatedProperties": false
         }
       },
-      "properties": {},
-      "type": "object"
+      "additionalProperties": false
     }
   },
-  "title": "stack",
-  "type": "object"
+  "additionalProperties": false
```

Comment:
We'll need to ensure blocking additional properties in all these places doesn't break anything